python-backend/app/api/now_applications/models/activity_detail/placer_operation_detail.py

```diff
@@ -10,7 +10,7 @@
 
 class PlacerOperationDetail(ActivityDetailBase):
     __mapper_args__ = {
-        'polymorphic_identity': 'placer',  ## type code
+        'polymorphic_identity': 'placer_operation', ## type code
     }
 
```

Comment:
@pdittaro I changed the name fo the seed data for activity_type_codes in the previous PR,  and missed two references